jina/optimizers/parameters.py

```diff
@@ -37,38 +44,36 @@ def __init__(
         self.log = log
 
 
-class FloatParameter(OptimizationParameter):
-    def __init__(
-        self,
-        low: float,
-        high: float,
-        step_size: Optional[float] = None,
-        log: bool = False,
-        *args,
-        **kwargs,
-    ):
-        super().__init__(*args, **kwargs)
-        self.low = low
-        self.high = high
-        self.step_size = step_size
-        self.log = log
-
-
 class UniformParameter(OptimizationParameter):
```

Comment:
why can't we use `uniform` sampling also for `Integers`?

Reply:
In some sense, it is what the normal integer sampling does. But you don't call it uniform sampling, but rather choosing a value with a certain step size. I would not try to invent the wheel a second time here, but go with what the people that seriously thought about hyper parameter optimization decided. We can change, if we see it does not fit well for us and we see user requests to change it. But for now, I wouldn't deem us smarter then the optuna folks.